### ISSUE TITLE
The latest stable version of the app crashes in Mac

### ISSUE THREAD

**Issue by deutranium** [xournalpp/xournalpp#2975]
**Affects versions :**
 - **OS:** Mac Big Sur
 - **Which version of libgtk do you use:** 3.24.10
 - **Version of Xournal++:** 1.0.20 (The one from "Download Stable Release" for Mac from the website
 - **Installation method:** Official Release

**Describe the bug**
Not able to use the app, it crashes as soon as it starts

**To Reproduce**
1. Download the app from the [website](https://xournalpp.github.io/installation/macos/) which linked to [this](https://github.com/xournalpp/xournalpp/releases/download/1.0.20/xournalpp-1.0.20-macos.zip)
2. Extract the zip
3. Click on the Xournal++ app
4. The app window will be seen for an instant and crashes immediately after without any warning

**Expected behavior**
Clicking on the app runs it properly

**Additional context**
The problem was solved by using Xournal++ version 1.0.19

**Comment by Febbe** [comment]
duplicate of #2836

**Comment by Febbe** [comment]
Tl;dr: We are aware of this issue, and we also have a fixup, so that the application does not crash.
Thank you for the issue.

**Comment by rolandlo** [comment]
The [nightly build](https://github.com/xournalpp/xournalpp/releases/tag/nightly) released tonight (or later) should fix the crash.
